DESCRIPTION

```diff
@@ -48,13 +48,17 @@ Depends:
 Imports:
     BBmisc (>= 1.11),
     backports (>= 1.1.0),
+    cowplot,
+    hrbrthemes,
     ggplot2,
     stats,
     stringi,
     checkmate (>= 1.8.2),
     data.table,
     methods,
     parallelMap (>= 1.3),
+    purrr,
+    sf,
```

Comment:
We try to keep the list of imports as short as possible -- it is absolutely necessary to add all of those packages here?

Reply:
`purrr` is needed for `imap` (that does not exist in any other package and neither in BBmisc)
`sf` does all the processing in the background of the spatial plotting in `ggplot2`

Reply:
But does it need to be in imports? Could it be in suggests?

Reply:
Both packages are used directly in `createResamplingPlots()`, so we need to have it in imports because I directly import it in the function? (Correct me if I'm wrong)

```
#' @import sf
#' @importFrom purrr map imap is_list flatten map_int
```

But I think I can move `cowplot` and `hrbrthemes` to suggests?

I also made some `importFrom` restrictions for `sf` and `hrbrthemes`.